tests/functional/deprecations/test_deprecations.py

```diff
@@ -306,17 +308,27 @@ def models(self):
     @mock.patch.dict(os.environ, {"DBT_ENV_PRIVATE_RUN_JSONSCHEMA_VALIDATIONS": "True"})
     def test_deprecated_invalid_deprecation_date(self, project):
         event_catcher = EventCatcher(GenericJSONSchemaValidationDeprecation)
+        note_catcher = EventCatcher(Note)
         try:
-            run_dbt(["parse", "--no-partial-parse"], callbacks=[event_catcher.catch])
+            run_dbt(
+                ["parse", "--no-partial-parse"],
+                callbacks=[event_catcher.catch, note_catcher.catch],
+            )
         except:  # noqa
             assert (
                 True
             ), "Expected an exception to be raised, because a model object can't be created with a deprecation_date as an int"
 
-        assert len(event_catcher.caught_events) == 1
-        assert (
-            "1 is not of type 'string', 'null' in file" in event_catcher.caught_events[0].info.msg
-        )
+        if GJVD()._is_preview:
```

Comment:
if `_is_preview` is being accessed outside of it's class isn't the leading underscore kind of a misnomer and it should just be `is_preview`?